README.md

```diff
@@ -1,2 +1,5 @@
+![Automation Library CI/CD](https://github.com/kmdlogic/kmd-logic-gateway-automation/workflows/Automation%20Library%20CI/CD/badge.svg?branch=master)
+![CLI Tool CI/CD](https://github.com/kmdlogic/kmd-logic-gateway-automation/workflows/CLI%20Tool%20CI/CD/badge.svg?branch=master)
+
 # kmd-logic-gateway-automation
-Kmd loci
+Kmd Logic gateway automation
```

Comment:
Upper case please

Reply:
Ahhh. whatever. It must be totally rewritten :D

Reply:
Already fixed :P But please see status badges! :)